### PR TITLE
docs(combine): add note about recycling array instance across emissions

### PR DESCRIPTION
**Pull request by appsforartists**
Closes https://github.com/staltz/xstream/issues/153

### CODE REVIEW

**Comment by appsforartists** [comment]
Took the same note from https://github.com/staltz/xstream/pull/155 and moved it to JSDoc

File: src/core.ts [hunk 1]
```diff
@@ -1618,6 +1618,15 @@ export class Stream<T> implements InternalListener<T> {
...
    *
+   * Note: to minimize garbage collection, *combine* uses the same array
+   * instance for each emission.  If you need to compare emissions over time,
+   * cache the values with `map` first:
+   *
+   * ```js
+   * xs.combine(stream1, stream2).map(
+   *   combinedEmissions => ([ ...combinedEmissions ])
+   * ).compose(pairwise)...
+   *
```
Comment (staltz) [inline src/core.ts]: Missing closing triple quotes here.

Comment (Widdershin) [inline src/core.ts]: Also the trailing `...` on line 1628 will cause `markdown-doctest` to fail. Could it be removed?

Comment (appsforartists) [inline src/core.ts]: Good catch.  Did this a bit too quickly.

Thanks!

**Comment by staltz** [comment]
Thanks

**Pull request merged**